springer-basic-author-date.csl

```diff
@@ -121,10 +113,16 @@
              In: Smith J (ed) The rise of modern genomics, 3rd edn.
              Wiley, New York, pp 230-257 -->
         <if type="chapter paper-conference" match="any">
-          <text macro="editor"/>
-          <group prefix=" " delimiter=", " suffix=". ">
-            <text variable="container-title"/>
-            <text macro="edition"/>
+          <text term="in" text-case="capitalize-first" suffix=": "/>
```

Comment:
It likely doesn't make a difference, but let's move the ```in``` term into the group (and them remove the space in the suffix, because you now have a delimiter). In the very unlikely case that we have neither a container title nor an editor, we likely don't want the "in", either.